felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/rapidsrivers/service/ServiceRiver.kt

```diff
@@ -132,41 +122,19 @@ sealed class ServiceRiver : ObjectRiver<ServiceMelding>() {
                 null
             }
 
-            // Støtter Key.DATA som flagg med datafelt på rot (metode på vei ut)
-            Key.DATA in json &&
-                (
-                    service.startKeys.all(json::containsKey) ||
-                        service.dataKeys.any(json::containsKey)
-                ) -> {
-                DataMelding(
-                    eventName = Key.EVENT_NAME.krev(service.eventName, EventName.serializer(), json),
-                    transaksjonId = Key.UUID.les(UuidSerializer, json),
-                    dataMap =
-                        json.filterKeys {
-                            (service.startKeys + service.dataKeys).contains(it)
-                        },
-                )
-            }
-
-            // Støtter Key.DATA som objekt som inneholder datafelt (metode på vei inn)
-            // TODO Når all data er nested under Key.DATA så kan startKeys og dataKeys få visibility protected
-            service.startKeys.all(nestedData::containsKey) ||
-                service.dataKeys.any(nestedData::containsKey) -> {
```

Comment:
Jeg går ut fra at konseptet med `startKeys` og `dataKeys` og den sjekken vi gjør når vi sjekker at meldingen enden har alle startkeys eller noen datakeys har blitt lagt til en gang i tiden for å få litt mer kontroll på hvilke meldinger som slipper gjennom som Datameldinger til den enkelte servicen (?). Er det sånn at vi ikke trenger denne kontrollen lenger?

Reply:
Denne kontrollen eksisterer nå i `lesStegX`-funksjonene i de ulike servicene. Da leser man verdier fra meldingen og finner hvilke verdier man har og hvilke man mangler. Den sjekken som blir fjernet her var ment å forhindre en melding med _flagget_ `Key.DATA`, men ingen verdier, å nå en service som så prøvde og lese verdiene.

Reply:
Det gir mening!